egs/commonvoice/ASR/local/word_segment_yue.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+# Copyright    2024  Xiaomi Corp.        (authors: Zengrui Jin)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script takes a text file "data/lang_char/text" as input, the file consist of
+lines each containing a transcript, applies text norm and generates the following
+files in the directory "data/lang_char":
+    - transcript_words.txt
+    - words.txt
+    - words_no_ids.txt
+"""
+
+import argparse
+import logging
+import re
+from pathlib import Path
+from typing import List
+
+import pycantonese
```

Comment:
Could you also document how this package is installed?

Reply:
This package can be installed directly using `pip install`, this package is also listed in the `requirements.txt`